### PR TITLE
Added CaseControl tool

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

File: biopet/sampleconfig.wdl [hunk 1]
```diff
@@ -77,3 +79,41 @@ task SampleConfigCromwellArrays {
...
+        ~{toolCommand} CromwellArrays \
+        -i ~{sep="-i " inputFiles} \
+        -s ~{sep="-s " inputFiles} \
+        ~{"-o " + outputPath} \
+        ~{"--controlTag " + controlTag}
+    }
+
+    output {
+        File outputFile = outputPath
+        Array[CaseControl] controls = read_json(outputFile)
```
Comment (johnmous) [inline biopet/sampleconfig.wdl]: Better name it caseControls instead of controls?

File: biopet/sampleconfig.wdl [hunk 2]
```diff
@@ -77,3 +79,41 @@ task SampleConfigCromwellArrays {
...
+        then "java -Xmx" + memory + "G -jar " + toolJar
+        else "biopet-sampleconfig -Xmx" + memory + "G"
+
+    command {
+        set -e -o pipefail
+        ~{preCommand}
+        mkdir -p $(dirname ~{outputPath})
+        ~{toolCommand} CromwellArrays \
+        -i ~{sep="-i " inputFiles} \
+        -s ~{sep="-s " inputFiles} \
```
Comment (johnmous) [inline biopet/sampleconfig.wdl]: Replace inputFiles with sampleConfigs

File: biopet/sampleconfig.wdl (hunk 1 above)
Comment (ffinfo) [reply biopet/sampleconfig.wdl]: fixed

File: biopet/sampleconfig.wdl (hunk 2 above)
Comment (ffinfo) [reply biopet/sampleconfig.wdl]: Good one, fixed now

**Pull request merged**